src/GitHub/Data/Request.hs

```diff
@@ -241,6 +242,14 @@ instance Hashable (SimpleRequest k a) where
              `hashWithSalt` ps
              `hashWithSalt` body
 
+instance Hashable W.EscapeItem where
```

Comment:
orphan instance  :/.

Could you make a PR to `http-types` so the instance is there. Alternatively inline code where instance is needed?

I won't accept PR with orphan instance.

Reply:
I should have done this. But now it have wrapped the type in a newtype, because I don't want to go through the whole process of having the right http-types version in stackage.